src/editors/autocompleteEditor.js

```diff
@@ -122,12 +122,9 @@ class AutocompleteEditor extends HandsontableEditor {
             match = cellValue.substr(indexOfMatch, query.length);
             cellValue = cellValue.replace(match, `<strong>${match}</strong>`);
           }
-
-          TD.innerHTML = sanitize(cellValue);
-        } else {
-          TD.innerHTML = cellValue;
         }
 
+        TD.innerHTML = sanitize(cellValue);
```

Comment:
Should we block `allowHtml` completely?
Maybe we should leave making a decision for an end-developer?
We mentioned a potential risk in the documentation.
What do you think?

Reply:
IMO yes. Other HTML-like modules are sanitized in the same way (eq. [renderers](https://github.com/handsontable/handsontable/blob/develop/src/renderers/htmlRenderer.js#L17), ContextMenu). Why should the editor be treated differently?

Reply:
... OR maybe we shouldn't sanitize the features which directly informs the developers that it allows HTML to render. So the `allowsHtml: true` option and HTML renderer wouldn't filter the values at all? In this shape, sanitized values transform the upcoming release 8.2.0 to be potently Braking Change. As mentioned in the PR description the values won't render the custom tags (web components) and custom attributes anymore. 

cc @wojciechczerniak